app/src/main/java/com/kickstarter/ui/compose/designsystem/KSControls.kt

```diff
@@ -0,0 +1,223 @@
+
+            KSSwitch(checked = checkedOn, onCheckChanged = {}, enabled = false)
+
+            Spacer(modifier = Modifier.height(8.dp))
+
+            KSRadioButton(
+                selected = radioButtonSelected,
+                onClick = { radioButtonSelected = !radioButtonSelected }
+            )
+
+            Spacer(modifier = Modifier.height(8.dp))
+
+            KSRadioButton(
+                selected = !radioButtonSelected,
+                onClick = { radioButtonSelected = !radioButtonSelected }
+            )
+
+            Spacer(modifier = Modifier.height(8.dp))
+
+            KSRadioButton(selected = radioButtonSelected, onClick = {}, enabled = false)
+
+            Spacer(modifier = Modifier.height(8.dp))
+
+            KSCheckbox(checked = checked, onCheckChanged = { checked = !checked })
+
+            Spacer(modifier = Modifier.height(8.dp))
+
+            KSCheckbox(checked = checked, onCheckChanged = {}, enabled = false)
+
+            Spacer(modifier = Modifier.height(8.dp))
+
+            KSStringDropdown(
+                onItemSelected = { _, _ -> },
+                items = arrayOf("Coffee", "Soda", "Water")
+            )
+        }
+    }
+}
+
+@Composable
+fun KSSwitch(
+    checked: Boolean,
+    onCheckChanged: ((Boolean) -> Unit),
+    enabled: Boolean = true
+) {
+    Switch(
+        checked = checked,
+        onCheckedChange = onCheckChanged,
+        colors = SwitchDefaults.colors(
+            uncheckedThumbColor = colors.kds_support_100,
+            uncheckedTrackColor = colors.kds_support_500,
+            uncheckedTrackAlpha = 0.38f,
+            checkedThumbColor = colors.kds_create_700,
+            checkedTrackColor = colors.kds_create_700,
+            checkedTrackAlpha = 0.38f,
+            disabledCheckedThumbColor = colors.kds_support_300,
+            disabledCheckedTrackColor = colors.kds_support_700,
+        ),
+        enabled = enabled
+    )
+}
+
+@Composable
+fun KSRadioButton(
+    selected: Boolean,
+    onClick: () -> Unit,
+    enabled: Boolean = true
+) {
+    RadioButton(
+        selected = selected,
+        onClick = onClick,
+        colors = RadioButtonDefaults.colors(
+            selectedColor = colors.kds_create_700,
+            unselectedColor = colors.kds_create_700,
+            disabledColor = colors.kds_support_300
+        ),
+        enabled = enabled
+    )
+}
+
+@Composable
+fun KSCheckbox(
+    checked: Boolean,
+    onCheckChanged: (Boolean) -> Unit,
+    enabled: Boolean = true
+) {
+    Checkbox(
+        checked = checked,
+        onCheckedChange = onCheckChanged,
+        colors = CheckboxDefaults.colors(
+            checkedColor = colors.kds_create_700,
+            uncheckedColor = colors.kds_create_700,
+            checkmarkColor = colors.kds_white,
+            disabledColor = kds_support_500,
+        ),
+        enabled = enabled
+    )
+}
+
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+fun KSStringDropdown(
```

Comment:
A bit worried about the `ExperimentalMaterialApi` for dropdowns, but one of the first UI pieces I would like to kill and substitute over this component will be the Shipping rules dropdown. The usability will dramatically improve.

Reply:
Yeah this was the only one that needed it fortunately.  The non experimental one didn't seem to match what we needed so I opted for this, but if something better replaces it all the better